### DISCUSSION TITLE
Signle session

### DISCUSSION THREAD

**Discussion by Masharipov** [cvnlab/GLMsingle#131] [Q&A] [answered]
Hi! I have a rapid event-related design containing a single session (run) per subject with two conditions (100 trials per condition). Did I understand correctly that I cannot use fractional ridge regression to regularize the single-trial betas becouse it requeres several runs to calucluate optimal degree of regularization via cross-validation? I wanted to check how fractional ridge regression works compared to the least squares separate (LSS) approach (for task with single run).

**Comment by kendrickkay** [comment, marked as answer]
Yes, GLMsingle is designed to use cross-validation across runs (e.g. to set the ridge regression hyperparameter). One approach is to artificially split your run into two halves, and then use GLMsingle. That could work.

Regarding LSS, this is discussed in the GLMsingle eLife paper.  Basically, you can think of LSS as an extreme regularization approach. It may work well when SNR is really poor (since it treats all conditions other than the current trial as if they evoke the same response).  Ridge regression can viewed as a tunable regularization where the regularization is applied only if the data need it.
